shortfin/tests/apps/sd/serving_test.py

```diff
@@ -0,0 +1,186 @@
+# Copyright 2024 Advanced Micro Devices, Inc.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import requests
+import time
+import pytest
+import subprocess
+import os
+import socket
+import sys
+import copy
+from contextlib import closing
+import numpy
+from PIL import Image
+
+from shortfin_apps.types.Base64CharacterEncodedByteSequence import (
+    Base64CharacterEncodedByteSequence,
+)
+
+from shortfin_apps.utilities.image import (
+    image_from,
+)
+
+BATCH_SIZES = [1]
+
+sample_request = {
+    "prompt": [
+        " a cat under the snow with blue eyes, covered by snow, cinematic style, medium shot, professional photo, animal",
+    ],
+    "neg_prompt": ["Watermark, blurry, oversaturated, low resolution, pollution"],
+    "height": [1024],
+    "width": [1024],
+    "steps": [20],
+    "guidance_scale": [7.5],
+    "seed": [0],
+    "output_type": ["base64"],
+    "rid": ["string"],
+}
+
+
+def start_server(fibers_per_device=1, isolation="per_fiber"):
+    # Start the server
+    srv_args = [
+        "python",
```

Comment:
Prefer to use `sys.executable` instead of `python` to launch a python subprocess from another python script (that's more portable across operating systems and inside/outside of a venv)